src/components/icons/exports.js

```diff
@@ -0,0 +1,314 @@
+PinActive.displayName = 'PinActiveIcon';
+export const Pin = withThemeProp(withSizeProp(OrigPin));
+Pin.displayName = 'PinIcon';
+export const Plugin = withThemeProp(withSizeProp(OrigPlugin));
+Plugin.displayName = 'PluginIcon';
+export const Refresh = withThemeProp(withSizeProp(OrigRefresh));
+Refresh.displayName = 'RefreshIcon';
+export const Restore = withThemeProp(withSizeProp(OrigRestore));
+Restore.displayName = 'RestoreIcon';
+export const Revert = withThemeProp(withSizeProp(OrigRevert));
+Revert.displayName = 'RevertIcon';
+export const Review = withThemeProp(withSizeProp(OrigReview));
+Review.displayName = 'ReviewIcon';
+export const Rocket = withThemeProp(withSizeProp(OrigRocket));
+Rocket.displayName = 'RocketIcon';
+export const ScreenGear = withThemeProp(withSizeProp(OrigScreenGear));
+ScreenGear.displayName = 'ScreenGearIcon';
+export const ScreenUser = withThemeProp(withSizeProp(OrigScreenUser));
+ScreenUser.displayName = 'ScreenUserIcon';
+export const Search = withThemeProp(withSizeProp(OrigSearch));
+Search.displayName = 'SearchIcon';
+export const Sorting = withThemeProp(withSizeProp(OrigSorting));
+Sorting.displayName = 'SortingIcon';
+export const Speedometer = withThemeProp(withSizeProp(OrigSpeedometer));
+Speedometer.displayName = 'SpeedometerIcon';
+export const Split = withThemeProp(withSizeProp(OrigSplit));
+Split.displayName = 'SplitIcon';
+export const Star = withThemeProp(withSizeProp(OrigStar));
+Star.displayName = 'StarIcon';
+export const Success = withThemeProp(withSizeProp(OrigSuccess));
+Success.displayName = 'SuccessIcon';
+export const Support = withThemeProp(withSizeProp(OrigSupport));
+Support.displayName = 'SupportIcon';
+export const Switcher = withThemeProp(withSizeProp(OrigSwitcher));
+Switcher.displayName = 'SwitcherIcon';
+export const Table = withThemeProp(withSizeProp(OrigTable));
+Table.displayName = 'TableIcon';
+export const TagMulti = withThemeProp(withSizeProp(OrigTagMulti));
+TagMulti.displayName = 'TagMultiIcon';
+export const TagStacked = withThemeProp(withSizeProp(OrigTagStacked));
+TagStacked.displayName = 'TagStackedIcon';
+export const Tag = withThemeProp(withSizeProp(OrigTag));
+Tag.displayName = 'TagIcon';
+export const TreeStructure = withThemeProp(withSizeProp(OrigTreeStructure));
+TreeStructure.displayName = 'TreeStructureIcon';
+export const Truck = withThemeProp(withSizeProp(OrigTruck));
+Truck.displayName = 'TruckIcon';
+export const UserFilled = withThemeProp(withSizeProp(OrigUserFilled));
+UserFilled.displayName = 'UserFilledIcon';
+export const UserLinear = withThemeProp(withSizeProp(OrigUserLinear));
+UserLinear.displayName = 'UserLinearIcon';
+export const Verified = withThemeProp(withSizeProp(OrigVerified));
+Verified.displayName = 'VerifiedIcon';
+export const Warning = withThemeProp(withSizeProp(OrigWarning));
+Warning.displayName = 'WarningIcon';
+export const World = withThemeProp(withSizeProp(OrigWorld));
```

Comment:
Export names do change or I am lost: `import { WorldIcon } from ` won‘t work anymore.

Reply:
I'm not sure what you're confused about here. This was the all point of having the `Icon` suffix removed (which I only did because people were asking for it).